infra/examples-dev/aws-all/main.tf

```diff
@@ -34,11 +34,13 @@ module "worklytics_connectors" {
   source = "../../modules/worklytics-connectors"
   # source = "git::https://github.com/worklytics/psoxy//infra/modules/worklytics-connectors?ref=rc-v0.4.29"
 
-  enabled_connectors    = var.enabled_connectors
-  example_jira_issue_id = var.example_jira_issue_id
-  jira_cloud_id         = var.jira_cloud_id
-  jira_server_url       = var.jira_server_url
-  salesforce_domain     = var.salesforce_domain
+  enabled_connectors        = var.enabled_connectors
+  example_jira_issue_id     = var.example_jira_issue_id
```

Comment:
Disregard for this PR, just a general comment for future: but I see this growing and find it painful. To me, we should have a map (string, object)
```tf
enabled_connectors_settings = {
"jira": {
"jira_issue_id" : var.example_jira_issue_id
}
```
or
```tf
enabled_connectors_settings = {
jira_example_issue_id = var.example_jira_issue_id,
jira_cloud_id = ...
}
```
and maybe rely on local validators on what to expect

Reply:
https://app.asana.com/0/1201039336231823/1205054651201385